tests/MongoDB.EntityFrameworkCore.FunctionalTests/Storage/MongoClientWrapperTests.cs

```diff
@@ -178,6 +178,45 @@ public async Task CreateDatabase_creates_nested_index_on_owns_many_owns_one(bool
         Assert.Equal(3, indexList.Count);
     }
 
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
+    public async Task CreateDatabase_creates_alternate_keys(bool async)
```

Comment:
Should we also add tests for multiple alternate keys on the same entity?

Reply:
Good idea, now updated. Also made sure the names reflect any bson mapping in the test.